libnavigation-base/src/main/java/com/mapbox/navigation/base/route/NavigationRoute.kt

```diff
@@ -702,4 +702,9 @@ private class ByteBufferBackedInputStream(
         buffer.get(bytes, off, bytesToRead)
         return bytesToRead
     }
+
+    override fun close() {
+        buffer.rewind()
```

Comment:
Not necessary for the fix, but it puts the buffer into the original position, which is more clean and accurate IMO.